Lubandeo/Kotlin_Marvel_App
# Implemented recycler view - glide

## Description

Implemented recycler view to show the list of characters, each of them in a card view. Used Glide to show the images.
## Story Type

- New feature :white_check_mark:
- Change request
- Bug fix
- Tech
## How Can This Be Tested?

By pressing on the go to app button and scrolling down the screen once the characters appear to see them all 
## Evidences

| Pixel 2  | Nexus S|
| ------------- | -------------|
| ![gif cropped](https://user-images.githubusercontent.com/109096165/191522194-1528453c-6817-46a0-9366-97c7a0ac3830.gif) | ![gif cropped nexus](https://user-images.githubusercontent.com/109096165/191530333-32284b19-6d64-41aa-9166-89dfe49f172c.gif)|
## Checklist

## Tested On
- Pixel 2
- Nexus S
## Package Structure
<img width="227" alt="carpetas1" src="https://user-images.githubusercontent.com/109096165/191522329-c85ebd53-9e09-4e19-8f45-832e982a5957.png">
<img width="291" alt="image" src="https://user-images.githubusercontent.com/109096165/191522575-d8711a78-327d-4226-8687-34f649fdca0c.png">
## Test Coverage
<img width="388" alt="Mappers coverage - recycler " src="https://user-images.githubusercontent.com/109096165/191522714-5ea2e849-cf9a-4b22-b689-016a5cc64c35.png">
<img width="392" alt="mvvm coverage - recycler " src="https://user-images.githubusercontent.com/109096165/191522751-9f2b8c74-69cf-4d23-89ca-38b7af887819.png">
<img width="385" alt="usecase coverage - recycler" src="https://user-images.githubusercontent.com/109096165/191522788-2ba6558d-a1df-4530-aeb0-a78f67fc2c72.png">
